package.json

```
@@ -27,10 +27,10 @@
   },
   "homepage": "https://github.com/dashevo/DashJS#readme",
   "dependencies": {
-    "@dashevo/dapi-client": "~0.8.0-dev.14",
-    "@dashevo/dashcore-lib": "~0.18.0",
-    "@dashevo/dpp": "0.10.0-dev.14",
-    "@dashevo/wallet-lib": "5.0.3",
+    "@dashevo/dapi-client": "~0.9.0-dev.6",
+    "@dashevo/dashcore-lib": "^0.18.0",
+    "@dashevo/dpp": "^0.11.0",
```

Thinking we should do this to be safe...
```suggestion
    "@dashevo/dpp": "~0.11.0",
```